lib/Doctrine/DBAL/Schema/PostgreSqlSchemaManager.php

```diff
@@ -329,11 +328,9 @@ protected function _getPortableTableColumnDefinition($tableColumn)
             $autoincrement           = true;
         }
 
-        if (preg_match("/^['(](.*)[')]::.*$/", $tableColumn['default'], $matches)) {
+        if (preg_match("/^['(](.*)[')]::/", $tableColumn['default'], $matches)) {
             $tableColumn['default'] = $matches[1];
-        }
-
-        if (stripos($tableColumn['default'], 'NULL') === 0) {
+        } elseif (preg_match('/^NULL::/', $tableColumn['default'])) {
```

Comment:
This needs to be case-insensitive (it was `stripos()`).

Reply:
Does it? Looks like it's always upper-cased in the introspection output which is covered by a test. The existing implementation may be just too precautious, same as in https://github.com/doctrine/dbal/pull/2854#discussion_r255346946.

Reply:
I'm not sure (didn't check tests), if you are then :+1:, just from the diff you removed case-insensitive check. If same as https://github.com/doctrine/dbal/pull/2854#discussion_r255346946, then it's fine.